stories/slider.story.tsx

```diff
@@ -9,16 +9,20 @@ const wrapperStyle = css({
   width: 400
 });
 
+interface State {
+  value: [number, number];
+}
+
 storiesOf("Slider", module).add(
   "simple",
-  withState({ value: [10, 20] })(({ store }) => (
+  withState<State>({ value: [10, 20] })(({ store }) => (
     <Slider
-      onValuesUpdated={({ values }) => store.set({ value: values })}
+      onValuesUpdated={({ min, max }) => store.set({ value: [min, max] })}
```

Comment:
This was wrong.
Fixed in https://github.com/ritz078/pebble/pull/167/files#diff-221f70a125dc1788290163c421ba27a5R20